CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [1.5.0] - 2023.06.16
+
+### Added
+- Load balancer feature for Btc, Ltc, Doge, Eth, Flare.
```

Comment:
Adjust this - make it more exhaustive pls

Reply:
Added more info